examples/daal4py/sycl/decision_forest_regression_batch.py

```diff
@@ -33,16 +33,11 @@ def read_csv(f, c, t=np.float64):
         return np.loadtxt(f, usecols=c, delimiter=',', ndmin=2, dtype=t)
 
 try:
-    from dpctx import device_context, device_type
-    with device_context(device_type.gpu, 0):
+    from daal4py.oneapi import sycl_context
+    with sycl_context('gpu'):
         gpu_available = True
 except Exception:
-    try:
-        from daal4py.oneapi import sycl_context
-        with sycl_context('gpu'):
-            gpu_available = True
-    except Exception:
```

Comment:
Also `Exception` is only slightly better. We should catch what is raised and that's apparently a `RuntimeError`